templates/workspace_services/guacamole/guacamole-server/docker/services/oauth/run

```diff
@@ -21,4 +21,6 @@ cookiesecret=$(dd if=/dev/urandom bs=32 count=1 2>/dev/null | base64 | tr -d --
 --set-xauthrequest true \
 --pass-basic-auth true \
 --cookie-refresh 50m \
---scope "openid offline_access ${AUDIENCE}/user_impersonation"
+--insecure-oidc-allow-unverified-email true \
+--oidc-groups-claim "roles" \
```

Comment:
I think roles is not the one holding the groups info. Maybe we can do something like this: https://learn.microsoft.com/en-us/answers/questions/300079/how-to-get-group-information-as-part-of-claims-in.html. I'd be happy to look into this together if you'd like.

Reply:
Sure we can take it offline